httplog/httplog.go

```diff
@@ -37,6 +39,71 @@ type Logger struct {
 	mode Mode
 }
 
+type ModuleMode struct {
+	Module string
+	mode   Mode
+}
+
+type Config struct {
+	ModuleModes []ModuleMode
+	modules     []string
+}
+
+func (m *Config) ParseMode(s string) (ModuleMode, error) {
+	var mode Mode
+
+	module, modeStr, found := strings.Cut(s, ":")
```

Comment:
Standard naming is  `ok`.

Reply:
Sorry, I don't understand this comment - the function signature is 

```go
func Cut(s, sep string) (before, after string, found bool)
```

?

Reply:
Interesting, usually the when bool is returned it's called `ok`